test/file.ts

```diff
@@ -48,6 +48,7 @@ import {
   GetSignedUrlConfig,
   GenerateSignedPostPolicyV2Options,
   GenerateSignedPostPolicyV2Callback,
+  Storage,
```

Comment:
Is this import needed?

Reply:
Good catch. It must've autoimported when I was testing something and I missed it